### ISSUE TITLE
Add a new variable for External Script Editor Args 

### ISSUE THREAD

**Issue by Merichbier** [Unity-Technologies/com.unity.ide.vscode#28]
I'm using a multi-root workspace in some projects. Currently my External Script Editor Args looks like :
```
"$(ProjectPath)/unity.code-workspace" -g "$(File)":$(Line):$(Column)
```
This way the workspace is open correctly by unity. A not so nice drawback is that all vs code project windows are named "unity (workspace)".

It would be nice to have an additional variable like $(ProjectName) to allow the following External Script Editor Args :
```
"$(ProjectPath)/$(ProjectName).code-workspace" -g "$(File)":$(Line):$(Column)
```

**Comment by Chizaruu** [comment]
Erm, I don't know if you've seen my [fork ](https://github.com/Chizaruu/com.tsk.ide.vscode)but it has this feature added to it :)

![image](https://github.com/Unity-Technologies/com.unity.ide.vscode/assets/59434446/0c2b4cc4-7b49-4640-885d-57acc83c43e9)

**Comment by Merichbier** [comment]
> Erm, I don't know if you've seen my [fork ](https://github.com/Chizaruu/com.tsk.ide.vscode)but it has this feature added to it :)
> 
> ![image](https://github.com/Unity-Technologies/com.unity.ide.vscode/assets/59434446/0c2b4cc4-7b49-4640-885d-57acc83c43e9)

Yep saw it after having written this issue. Thanks fantastic fork !
